.travis.yml

```diff
@@ -1,15 +1,18 @@
 sudo: false
+
```

Comment:
these changes were automatically added when running tests. Not familiar with this file. Is this okay?